application/__init__.py

```diff
@@ -17,13 +17,21 @@
 #   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 #   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 #   IN THE SOFTWARE.
-
+import logging.config
+import yaml
 import os
 import sys
 
 from flask import Flask, Blueprint
 from werkzeug.serving import WSGIRequestHandler
 
+with open('logging_default_config.yaml', 'rt') as config_file:
+    config = yaml.safe_load(config_file.read())
+
+logging.config.dictConfig(config)
+logger = logging.getLogger(__file__)
```

Comment:
`__name__` which shows only the module (/submodule) names, is much less verbose than `__file__`. What do you think?
```suggestion
logger = logging.getLogger(__name__)
```

Reply:
Note in `__init__.py` (module entry) file of some module, the module name (e.g., in our case, "application") is shown.

Reply:
This is good as I didn't really understand the __name__ macro and I don't like verbosity as well. However, what if there are duplicates in names? For example, application/features and application/routes both contain a file named vnc.py(one uppercase and one lowercase), for clarity in the future do we still want it?

Reply:
In a single application, no two module can have the same name, right? i.e., no two files can be placed under the same path